packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -5,8 +5,8 @@
 %global foreman_max_version 3.11
```

Comment:
you need to bump this (and the line above)

Reply:
Will do, thanks. That's interesting we're bumping the minimum to 3.11.